### ISSUE TITLE
Add xtream IPTV account information

### ISSUE THREAD

**Issue by heberjeur** [oxyroid/M3UAndroid#106]
**Is your feature request related to a problem? Please describe.**
Currently, the M3U application lacks support for adding xtream IPTV account information, limiting user accessibility to their preferred IPTV content.

**Describe the solution you'd like**
I would like to propose the addition of a feature that allows users to input their xtream IPTV account information directly into the M3U application, enabling seamless integration and access to IPTV content.

**Describe alternatives you've considered**
I suppose to be like this ..
![Screenshot_20240306-191047](https://github.com/realOxy/M3UAndroid/assets/11852634/72e59eb5-cd97-4ccd-b5d3-f74aa341e093)

**Additional context**
This feature enhancement would greatly benefit users who rely on xtream IPTV services for their entertainment needs. By including support for xtream IPTV account information, the M3U application can become a more comprehensive and versatile solution for accessing IPTV content.

**Comment by themodfatherinc** [comment]
are you just looking for the ability to display the user info responses from the xtream server? i am working on this now but tbh i am slightly confused by the way the api calls are made in this code as a very new programmer. i am writing this using my own api calls while i figure it out, perhaps the developer can help implement it

**Comment by oxyroid** [comment]
Since this issue has a low priority, I have not started to work on it. 
You can clone this repository, push your code, and then we can continue to discuss your confusion.

**Comment by oxyroid** [comment]
Completed, Available in SNAPSHOT soon.

**Comment by heberjeur** [comment]
> Completed, Available in SNAPSHOT soon.

I will try it 😁
